newrelic/hooks/mlmodel_sklearn.py

```diff
@@ -189,6 +189,43 @@ def instrument_sklearn_ensemble_hist_models(module):
     _instrument_sklearn_models(module, model_classes)
 
 
+def instrument_sklearn_cluster_models(module):
+    model_classes = (
+        "AffinityPropagation",
+        "Birch",
+        "DBSCAN",
+        "MeanShift",
+        "OPTICS",
+    )
+    _instrument_sklearn_models(module, model_classes)
+
+
+def instrument_sklearn_cluster_agglomerative_models(module):
+    model_classes = (
+        "AgglomerativeClustering",
+        "FeatureAgglomeration",
```

Comment:
These exist in older versions:
```suggestion
        "FeatureAgglomeration",
        "Ward",
        "WardAgglomeration",
```

Reply:
I believe Ward and WardAgglomeration are models that are outside of our supportability window of 3 years

Reply:
Never mind on this change. ^ These are not in version 0.22.0 which I believe is the 3 year old version of this library and outside our support window.